### PR TITLE
Update README and LICENSE

### PR DESCRIPTION
**Pull request by seanofw**
Hi!  So I sent you guys an e-mail about this, but I figured a pull request was another good way to get in touch with you, in case the e-mail showed up in your spam folders.

You've all done good work bringing NBBC into the modern age, and since you seem to want to be officially responsible for it, I thought I'd give you my blessing in a properly Git fashion by submitting a pull request that updates the README and LICENSE to show the changing of the guard.

We should talk, I'm sure.  There may be things you want to do in NBBC that would be easier with my help, and perhaps there are even breaking changes that would make working with NBBC easier in a modern PHP world.  I haven't done a lot of PHP in the last (*cough*) seven years or so, but I'd be happy to talk with you guys about its future and maybe even to help you create that future.

So this pull request contains an updated README and LICENSE to reflect the reality of the situation, which is that my copy on SourceForge was old and stale, that you've got the latest-and-greatest version on GitHub, and that I'm happy to see NBBC continuing to be maintained by somebody who really wants to maintain it.

### CODE REVIEW

**Comment by linc** [comment]
Howdy Sean, thanks for reaching out. My email is lincoln at vanillaforums.com if you want to contact me directly. I haven't heard from anyone else on the team about this so I'm not sure where that email went. These updates & improvements look good to me. 

Our general contributing guidelines are over here (it's all pretty standard open source stuff): http://docs.vanillaforums.com/developer/contributing/ Note that we don't enforce the Contributor's Agreement for libraries like this for which we don't own the initial copyright anyway (since that would be pointless).

Did you have particular things you wanted to know or request? We don't currently have any specific plans for this library beyond maintenance and patches. As some background, this is one of 4 parsers available in Vanilla, so our user base is a bit diverse (the others are WYSIWYG,  HTML, and our default of Markdown) and we do some additional "magic" parsing for Vanilla features on the other side of them.

**Review by linc — APPROVED** [review]

**Comment by seanofw** [comment]
Sorry, I didn't get your message about this:  For whatever reason, I didn't get anything in my e-mail inbox or any notifications on GitHub until you merged it in.  I'm going to have to check my GitHub notification settings :(

I sent the previous e-mail to 'todd at vanillaforums.com', and to 'alexandre.c at vanillaforums.com'.  I made a best guess as to the committers who'd most frequently committed to the repo and sent the message to them, but apparently it wasn't the best guess.

Anyway, I don't have any specific things that I want or have a request for, although eventually it'd be nice to eventually move the documentation out of SourceForge and into a github.io page that hangs off the repo.  But I just mainly wanted to make sure that I'm available in case you guys need me, or in case you have architectural questions, or in case there's some weird glitch you can't resolve, or in case there's something you want to change and you're really not sure how to go about it.  It's been a while since I was in this code, but I know how it works, how it was _intended_ to work, and can be available for questions when they arise.

Should you need to contact me directly, send an e-mail to 'sean at werkema.com'; I check that multiple times per day, so the message will get to me pretty fast.

As a matter of curiosity, do you have any indication how much this is being used?  Or is it just offered as an available option?

**Comment by linc** [comment]
Sounds good, thanks.

I don't have any statistics, but anecdotally I'd estimate something like 20-40% of Vanilla installs use BBCode. As of Vanilla 2.3 (I think), this is the native BBCode parser. Previously it was an optional addon to enhance our native parser (which was quite bad, really).

**Pull request merged**
